packages/shared/watchPausable/index.ts

```diff
@@ -10,12 +10,13 @@ export interface WatchPausableReturn extends Pausable {
 
 export type WatchPausableOptions<Immediate> = WatchWithFilterOptions<Immediate> & PausableFilterOptions
 
-/**
- * @deprecated This function will be removed in future version.
- */
+/** @deprecated This function will be removed in future version. */
```

Comment:
In the derecated message, we better always provide solutions like "use X instead" or use "Vue's native X" etc.